test/specs/utils/api.js

```diff
@@ -13,120 +13,104 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
+import { setUpUtilAPI } from '../../steps/setup';
 import * as given from '../../steps/1_given';
 import * as when from '../../steps/2_when';
 import * as then from '../../steps/3_then';
 
 describe('API util', () => {
+	beforeEach(setUpUtilAPI);
 	Given(
-		'a config with "api.network" set to "main"',
-		given.aConfigWithAPINetworkSetTo,
+		'a config with api.network and api.nodes set to "main" and "http://localhost:4000"',
```

Comment:
I think `'a config with api.network set to "main" and api.nodes set to "http://localhost:4000"'` is less confusing.